lib/apiservers/engine/network/utils.go

```diff
@@ -564,7 +564,7 @@ func PortForwardingInformation(vc *viccontainer.VicContainer, ips []string) []ty
 	//c := cache.ContainerCache().GetContainer(cid)
 
 	if vc == nil {
-		log.Errorf("Could not find container with ID %s", vc.ContainerID)
+		log.Errorf("No container provided")
```

Comment:
I presume the context that was in `vc.ContainerID` that we're no longer logging here is the same as the variable `t.ContainerConfig.ContainerID` at the top of the diff? If it is not, is there a reason we're eliminating this context from the logs?

Reply:
How can you log vc.ContainerID if vc is nil? :)

Reply:
LOL good question 😆

Reply:
NEVERMIND